tests/phpunit/integration/Modules/Reader_Revenue_ManagerTest.php

```diff
@@ -231,6 +232,92 @@ function ( $domain ) {
 		$this->assertEquals( $expected_filter, urldecode( $filter ) );
 	}
 
+	public function test_sync_publication_onboarding_state_returns_empty_object() {
```

Comment:
This test name is not so useful as it simply describes the return value of the request, rather than the scenario that's being tested. Please can you make it more descriptive of the actual scenario.

This might illustrate the need for some additional tests to cover scenarios which are not currently tested.